ffs/scheduler/scheduler.go

```diff
@@ -272,15 +267,13 @@ func (s *Scheduler) resumeStartedDeals() error {
 		if err != nil {
 			return fmt.Errorf("getting resumed queued job: %s", err)
 		}
-		wg.Add(1)
 		go func(j ffs.Job) {
-			defer wg.Done()
+			log.Infof("resuming job %s with cid %s", j.ID, j.Cid)
 			// We re-execute the pipeline as if was dequeued.
 			// Both hot and cold storage can detect resumed job execution.
 			s.executeQueuedJob(j)
 		}(j)
 	}
-	wg.Wait()
```

Comment:
Resume deals and don't wait for them, keep evaluating new Queued Jobs. (Just as if they were pushed in the usual case)